library/network/src/modules/SuSEFirewall.rb

```diff
@@ -31,6 +31,8 @@
 #
 # Module for handling SuSEfirewall2.
 require "yast"
+require "network/firewalld"
+include Firewalld
```

Comment:
Please do not `include` modules in the global namespace, and not in the Yast module either. They would become visible to too much of existing code, possibly creating conflicts. It should be fine to include it in `SuSEFirewalld`.